src/test/java/com/stripe/model/ExpandableFieldDeserializerTest.java

```diff
@@ -0,0 +1,57 @@
+package com.stripe.model;
+
+import com.google.gson.Gson;
+import com.stripe.BaseStripeTest;
+import com.stripe.net.APIResource;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.gson.reflect.TypeToken;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class ExpandableFieldDeserializerTest extends BaseStripeTest {
+
+	private static Gson gson = APIResource.GSON;
+
+	private class TestObject {
```

Comment:
Perfect — using a simplified test type instead of one of our more complex actual ones is exactly the right thing to do.